install.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+# INSTALL DDB ON WSL WITH AN OFFICIAL UBUNTU VERSION
+
+# Check if the script is run as root if not rerun as root
+if [ "$(id -u)" != "0" ]; then
+   sudo $0
+    exit $?
+fi
+DDB_USER=${DDB_USER:-$SUDO_USER}
+
+if [ -z "$DDB_USER" ]; then
+    echo "DDB_USER is not defined"
+    exit 1
+fi
+
+#Update and upgrade packages
+apt-get update -y
+apt-get upgrade -y
+
+#Install and configuration of Docker
+## Add Docker's official GPG key:
+apt-get update -y
```

Comment:
You've litterally just done it, no need to do it again.

Reply:
you're right, corrected